### DISCUSSION TITLE
Reviews and Landing

### DISCUSSION THREAD

**Discussion by djmitche** [taskchampion/taskchampion#243] [General]
This is my usual process, but I'm happy to choose another.  In any case, we should agree:

* Someone submits a PR, and either requests a specific reviewer or reviewers are auto-assigned
* A reviewer reviews that PR
* If the submitter does not have write permission, the reviewer merges the PR; otherwise, the expectation is that the submitter will merge it

That last point is more important in things that deploy on push, since it means the submitter is around if the deployment fails.  But I prefer it even in cases like TaskChampion because the submitter can control when the change lands, manage conflicts with other PRs, and so on.

To the first point, I think we should set up CODEOWNERS.  Based on reviews so far, I think that would mean both @dbr and me, and GitHub will select whichever one of us is not the submitter.  Sound good?

**Comment by djmitche** [comment]
(I was reminded of this because I didn't merge https://github.com/taskchampion/taskchampion/pull/242#pullrequestreview-665423725)

**Reply by dbr** [reply]
Sounds good to me; I was actually deferring merging #242 to ask exactly this question!
